inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -527,6 +527,64 @@ void GNAGraphCompiler::finalizeConvolution1DPrimitive(InferenceEngine::CNNLayerP
 
 #if GNA_LIB_VER == 2
 
+void CheckCnn2DInputSupported(const unsigned h, const unsigned w) {
```

Comment:
Maybe it would be better to keep using uint_*t types as in the remainder of the file, just to stay aligned?
You could also make the new functions static.
Besides that maybe it's worth considering to move/add some definitions to gna_limitations.hpp.

Reply:
switched to uint_*tm
moved to gna_limitations.hpp and newly created gna_limitations.cpp.